.npmrc

```diff
@@ -0,0 +1,8 @@
+# More options: https://docs.npmjs.com/misc/config
+
+# Dependencies saved to package.json will be configured with an exact version
+# rather than using npm's default semver range operator.
+save-exact=true
```

Comment:
Why this? I don't think I saw it in other projects but I may be mistaken

Reply:
I copied it from a fury generated project